cylc/uiserver/handlers.py

```diff
@@ -97,7 +98,7 @@ def is_token_authenticated(handler: 'CylcAppHandler') -> bool:
     In these cases the bearer of the token is awarded full privileges.
     """
     identity_provider: JPSIdentityProvider = (
-        handler.serverapp.identity_provider
+        handler.serverapp.identity_provider  # type: ignore
```

Comment:
It is better to be specific about which error we are ignoring, so that other valid errors can still be flagged
```suggestion
        handler.serverapp.identity_provider  # type: ignore[union-attr]
```